### ISSUE TITLE
republish with newest openssl

### ISSUE THREAD

**Issue by romh87** [nginxinc/docker-nginx-unprivileged#112]
### Is your feature request related to a problem? Please describe

Latest openssl vulnerabilities solved in version 3.0.7

### Describe the solution you'd like

openssl 3.0.7 published yesterday, is already available in apk, please republish the dockerhub images so that we have the latest safest version

**Comment by romh87** [comment]
Good afternoon, any alpine image would be great, thank you

**Comment by romh87** [comment]
You are right, so version "OpenSSL 1.1.1s  1 Nov 2022 (Library: OpenSSL 1.1.1q  5 Jul 2022)" is solving the same issues as the mentioned 3.0.7

**Comment by thresheek** [comment]
No, it does not solve those vulnerabilities, because openssl 1.1 does not have the offending code. See https://www.openssl.org/news/secadv/20221101.txt for more details.

**Comment by romh87** [comment]
You are correct, 1.1.1s contains other bugs. I will close the issue, thanks for the explanations

**Comment by sbouchex** [comment]
@romh87 Alpine image is embedding "BusyBox v1.35.0" which is including unfixed openssl 3.0.x

Alpine 3.16.3 is fixing this issue and the nginxinc/nginx-unprivileged images must therefore be regenerated

**Comment by alessfg** [comment]
It's not yet been fixed in the upstream Alpine 3.16.3 image...

```
$ docker run -it --rm alpine sh
/ # apk add busybox
fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/main/aarch64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/community/aarch64/APKINDEX.tar.gz
OK: 5 MiB in 14 packages
/ # busybox | head -1
BusyBox v1.35.0 (2022-08-01 15:14:44 UTC) multi-call binary.
/ # cat /etc/alpine-release
3.16.3
```
